scripts/src/main/resources/scripts/command/az

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doSetup() {
+  if command -v az &> /dev/null
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then
+      doEcho "Azure CLI is already installed at $(command -v az)"
+    fi
+  else
+    doRequireNotBatch
+    if doIsWindows
+    then
+      # Get leatest release
+      if [ -z "${AZURECLI_VERSION}" ]
+      then
+        doEcho "Getting latest release..."
+        AZURECLI_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
+      fi
+      local version="${AZURECLI_VERSION}"
+      doDownload "-" "" "az" "${version}"
```

Comment:
Further, `doDownload` by default downloads and installs to the `software/«TOOL»` folder that is undesired here.
Analog to @CREITZ25 you should use a tool folder inside `update/install` instead.

Reply:
I assume that you mean `doInstall` instead of `doDownload`. I have changed the command and adressed myself to @CREITZ25 implementation of aws command.